### PR TITLE
picard: add optional qtmultimedia dependency

### PR DESCRIPTION
**Pull request by doronbehar**
###### Motivation for this change

~~Enable to~~ Fix this harmless error on startup:

```
W: 11:34:59,057 /nix/store/nhrp6qwqgvk34dvdrm7icalwqdi2h9a2-picard-2.3.1/lib/python3.7/site-packages/picard/ui/playertoolbar.__init__:106: Internal player: unavailable, cannot import name 'QtMultimedia' from 'PyQt5' (/nix/store/y5q9hfgg7pk9khgd0q45bms7gmk9nvnx-python3.7-PyQt5.sip-4.19.18/lib/python3.7/site-packages/PyQt5/__init__.py)
```

And make the integrated media player work.

###### Things done

- [x] Added new `enablePlayback` flag set by default to `false` which adds `qt5.qtmultimedia` to `nativeBuildInputs` and "chooses" the right pyqt5 build flavor.
- [x] Added a `passthru` for `pyqt5` that should be useful to easily indicate packages including it that it has (e.g) `multimediaEnabled`.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/picard/default.nix [hunk 1]
```diff
@@ -1,7 +1,12 @@
-{ stdenv, python3Packages, fetchFromGitHub, gettext, chromaprint, qt5 }:
+{ stdenv, python3Packages, fetchFromGitHub, gettext, chromaprint, qt5, enablePlayback ? false }:
```
Comment (doronbehar) [inline pkgs/applications/audio/picard/default.nix]: I'm not sure whether it should default to `true`..

Comment (ehmry) [reply pkgs/applications/audio/picard/default.nix]: I'm for enable-by-default, but I don't see any playback features in the new build or the changelog.

Comment (doronbehar) [reply pkgs/applications/audio/picard/default.nix]: TBH, now I notice this feature doesn't seem to work yet, but it does exist:

NUX-selection:2020 03 27-13:54:15

See the player at the bottom.

Also, launching `picard` with `--debug` gives:

```
...
D: 14:05:23,792 /nix/store/fgfg1gy5mwigm7a4h67vqv3plhax11q1-picard-2.3.1/lib/python3.7/site-packages/picard/ui/playertoolbar.__init__:91: Internal player: QtMultimedia available, initializing QMediaPlayer
D: 14:05:23,800 /nix/store/fgfg1gy5mwigm7a4h67vqv3plhax11q1-picard-2.3.1/lib/python3.7/site-packages/picard/ui/playertoolbar.__init__:98: Internal player: available, QMediaPlayer set up
...
```

I tested both latest version and `master` and I opened upstream bug report: https://tickets.metabrainz.org/browse/PICARD-1794 .

Comment (doronbehar) [reply pkgs/applications/audio/picard/default.nix]: That got fixed now thanks to missing gstreamer plugins I added as well.

File: pkgs/applications/audio/picard/default.nix [hunk 2]
```diff
@@ -13,7 +22,17 @@ in pythonPackages.buildPythonApplication rec {
     sha256 = "0xalg4dvaqb396h4s6gzxnplgv1lcvsczmmrlhyrj0kfj10amhsj";
   };
 
-  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ];
+  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ]
```
Comment (jonringer) [inline pkgs/applications/audio/picard/default.nix]:
```suggestion
  nativeBuildInputs = [ gettext qt5.qtbase ]
```

File: pkgs/applications/audio/picard/default.nix [hunk 3]
```diff
@@ -13,7 +22,17 @@ in pythonPackages.buildPythonApplication rec {
...
+    ++ stdenv.lib.optionals (pyqt5.multimediaEnabled) [
+      qt5.qtmultimedia.bin
+      wrapGAppsHook
```
Comment (jonringer) [inline pkgs/applications/audio/picard/default.nix]:
```suggestion
```

**Review by jonringer — COMMENTED** [review]
double wrapping still occuring:
```
[nix-shell:/home/jon/.cache/nixpkgs-review/pr-83482]$ ./results/picard/bin/
picard                    .picard-wrapped           ..picard-wrapped-wrapped
```
unfortunately, I don't think the python wrapper accepts arguments yet...

File: pkgs/applications/audio/picard/default.nix (hunk 3 above)
Comment (doronbehar) [reply pkgs/applications/audio/picard/default.nix]: Thanks for catching that @jonringer , I had this left over from tests I made with other environmental variables. Only this one is unneeded and without double wrapping doesn't occur - fixed in last push.

File: pkgs/applications/audio/picard/default.nix (hunk 2 above)
Comment (doronbehar) [reply pkgs/applications/audio/picard/default.nix]: However, this wrap hook is still needed for the many qt plugins environments.

**Comment by doronbehar** [comment]
Ping @timokau .

**Pull request merged**